start.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
 ./init_db.sh
-python oauth2server/manage.py loaddata oauth2server/apps/credentials/fixtures/ons_credentials.json
-python oauth2server/manage.py runserver 0.0.0.0:8040
\ No newline at end of file
+pipenv run oauth2server/manage.py loaddata oauth2server/apps/credentials/fixtures/ons_credentials.json
```

Comment:
I would leave these as `python...` as it's called from the Dockerfile (with system-installed deps).